test/e2e/libpod_suite_test.go

```diff
@@ -21,6 +21,11 @@ import (
 	"github.com/projectatomic/libpod/pkg/inspect"
 )
 
+const dockerRegistry = `
```

Comment:
Can you please call this a container registry. and could we use quay.io instead?

Reply:
I think leaving docker.io would be good, but I'd add quay.io as Dan suggested and put it in the first spot on the list.

Reply:
@rhatdan i can change the name; but this is being commented on out of context.  There are other configs for searching multiples.  And we dont pull from quay except in specific tests so there is no need for it to be there.